.circleci/config.yml

```diff
@@ -44,11 +46,11 @@ jobs:
       # database in neighbor container need some time to start
       # wait for the DB to be reachable, up to 15 pings with 1 second interval
       - run: |
-          set +eo pipefail;
+          set +o pipefail
```

Comment:
I don't see many pipes in this (or any other) snippet... I believe this can be removed as well.